scripts/run_integration_tests_aarch64.sh

```diff
@@ -50,6 +50,14 @@ update_workloads() {
         popd
     fi
 
+    FOCAL_OS_VHD_IMAGE_NAME="focal-server-cloudimg-arm64-custom.vhd"
+    FOCAL_OS_VHD_IMAGE="$WORKLOADS_DIR/$FOCAL_OS_VHD_IMAGE_NAME"
+    if [ ! -f "$FOCAL_OS_VHD_IMAGE" ]; then
+        pushd $WORKLOADS_DIR
+	time qemu-img convert -p -f raw -O vpc -o subformat=fixed $FOCAL_OS_RAW_IMAGE_NAME $FOCAL_OS_VHD_IMAGE_NAME || exit 1
+        popd
+    fi
+
```

Comment:
It would be so much nicer to do this in the test itself (I know this is how the QCOW2 is done too.) But I would like to see us do it in the test itself so we can reduce the complexity of the script and also only do it when needed.

Reply:
But isn't it better to do this only once, as we might want to run multiple tests using VHD?

Reply:
You can still test if it's present. And we only have one QCOW2 and one VHD test and I can't realistically see a need for more.